[2024-03-01 09:40:30,022] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: pythonoperator.t1 2024-03-01T09:40:28.559659+00:00 [queued]>
[2024-03-01 09:40:30,048] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: pythonoperator.t1 2024-03-01T09:40:28.559659+00:00 [queued]>
[2024-03-01 09:40:30,048] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-01 09:40:30,049] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-01 09:40:30,049] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-01 09:40:30,060] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): t1> on 2024-03-01T09:40:28.559659+00:00
[2024-03-01 09:40:30,066] {standard_task_runner.py:52} INFO - Started process 142 to run task
[2024-03-01 09:40:30,071] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pythonoperator', 't1', '2024-03-01T09:40:28.559659+00:00', '--job-id', '272', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pythonoperator.py', '--cfg-path', '/tmp/tmpnyp235d8', '--error-file', '/tmp/tmpky057pfg']
[2024-03-01 09:40:30,072] {standard_task_runner.py:77} INFO - Job 272: Subtask t1
[2024-03-01 09:40:30,135] {logging_mixin.py:109} INFO - Running <TaskInstance: pythonoperator.t1 2024-03-01T09:40:28.559659+00:00 [running]> on host b6da4fb5c714
[2024-03-01 09:40:30,209] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pythonoperator
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2024-03-01T09:40:28.559659+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-01T09:40:28.559659+00:00
[2024-03-01 09:40:30,259] {python.py:151} INFO - Done. Returned value was: None
[2024-03-01 09:40:30,274] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=pythonoperator, task_id=t1, execution_date=20240301T094028, start_date=20240301T094030, end_date=20240301T094030
[2024-03-01 09:40:30,323] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-01 09:40:30,354] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
